torch/onnx/symbolic_opset13.py

```diff
@@ -462,6 +463,50 @@ def reduce_dim(g, self, dim, keepdim, dtype):
     return reduce
 
 
+@_onnx_symbolic("aten::unflatten")
+@_beartype.beartype
+def unflatten(g: jit_utils.GraphContext, input, dim, unflattened_size):
```

Comment:
@justinchuby @titaiwangms Should we mention something like "implementation ported from torchlib + link"?

Reply:
I’m fine either way. Any considerations?

Reply:
Just a random thought, mostly for awareness. But it could get outdated 🤷

Reply:
A comment referencing a link to onnx Function?

Reply:
Added a comment with link and function name to point out the source.